admin_guide/upgrading_citus.rst

```diff
@@ -19,36 +22,34 @@ The others are known pg_upgrade manual steps, i.e. manually updating configurati
 
 We discuss the step by step process of upgrading the cluster below. Please note that you need to run the steps on all the nodes in the cluster. Some steps need to be run only on the master and they are explicitly marked as such.
 
-**1. Download and install Citus 5.0 on the server having the to-be-upgraded 4.0 data directory**
+**1. Download and install Citus 5 on the server having the to-be-upgraded 4.0 data directory**
 
-Download and install PostgreSQL 9.5.x from http://www.postgresql.org/download/.
+Follow the :ref:`installation instructions<requirements>` appropriate for your situation (e.g. single-machine or multi-machine). These instructions use the Citus package for your operating system, which includes PostgreSQL 9.5 as a dependency.
 
 .. note::
     The instructions below assume that the PostgreSQL installation is in your path. If not, you will need to add it to your PATH environment variable. For example:
-    
+
     ::
-        
-        export PATH=/usr/lib/postgresql/9.5/:$PATH
 
-You can then download and install the new Citus packages from our Downloads page. Please visit the :ref:`production` section for specific instructions.
+        export PATH=/usr/lib/postgresql/9.5/:$PATH
 
-Note that the Citus 5.0 extension will be installed at the PostgreSQL install location.
+Note that the Citus 5 extension will be installed at the PostgreSQL install location.
 
 **2. Setup environment variables for the data directories**
 
 Please set appropriate values for data location like below. This makes accessing data directories in subsequent commands easier.
 
 ::
 
-    export PGDATA5_0=/usr/lib/postgresql/9.5/data
-    export PGDATA4_0=/opt/citusdb/4.0/data
+    # this location differs by operating system
+    export PGDATA5=/usr/lib/postgresql/9.5/data
+
+    export PGDATA4=/opt/citusdb/4.0/data
 
 
 **3. Stop loading data on to that instance**
 
-If you are upgrading the master, then you should stop all data-loading/appending
-and staging before copying out the metadata. If data-loading continues after step 4 below,
-then the metadata will be out of date.
+If you are upgrading the master, then you should stop all data-loading/appending and staging before copying out the metadata. If data-loading continues after step 4 below, then the metadata will be out of date.
 
 **4. Copy out pg_dist catalog metadata from the 4.0 server (Only needed for master)**
```

Comment:
Might want to replace 4.0 with 4 to be more consistent with the rest of the document